### ISSUE TITLE
Cannot get to flash Mode

### ISSUE THREAD

**Issue by jpakusch** [klausahrenberg/WThermostatBeca#227]
Hi Folks,
I already jumped onto this issue: https://github.com/klausahrenberg/WThermostatBeca/issues/197
But there was no reply.
Sadly, i still did not get it to working.
I even bought another Device but got the same problem.
My original comment was:

_>Hi Everybody,
>Can i Jump aboard this topic as well?
>I have trouble flashing too using an BHT-3000GALW
>What i tried:
>exactly following your soldering guide using an esp Programmer, you already guessed it, i was not able to get to flash mode.
>So, i used the exact same setup with an ESP i had lying around (same pin layout). worked flawlessly.
>Now, i desoldered the onboard ESP and soldered mine to the Board. Assembled the Device, put it to 230V, nothing Burst into >Flames. So far, so good. But sadly, no AP containing "Thermostat" or anything similar appears.
>Pushing the up arrow key, as this was the procedure in default configuration of my device did not help either.
>Afterwards, i tried to flash the ESP the device came with outside of the device, just the naked ESP(inspired by this Thread), this >did not work either.
>Any Ideas?
>Of course my soldering could contain bugs or have destroyed something. But can i try something else to get the AP to show up?
>Why cant the old ESP be flashed? not even outside of the device?
>thank you for your help!_

i read these topics:
https://github.com/klausahrenberg/WThermostatBeca/issues/218
https://github.com/klausahrenberg/WThermostatBeca/issues/116
https://github.com/klausahrenberg/WThermostatBeca/issues/197

Summing up:

- The ESP contained in the device cannot be flashed, not soldered to the thermostat, not ouside of it
- Another "fresh" ESP can easily be flashed with the Software.
- When the ESP is soldered to the Thermostat, no wifi AP is opened.
- i tried the BHT-3000GALW and the Avatto me102.

Has anyone any idea i can try?
What about the baudrate? in the "Flashing problems and tips"-Issue there was some discussion about strange baurate settings. i tried nearly all of them, without any success....

Thank you very much for any assistance!
